resources/views/dashboard/widgets/previews/_members.blade.php

```diff
@@ -32,16 +32,16 @@
         @endforeach
     </div>
 @else
-    <div class="grid grid-cols-2 gap-2 members">
-        <?php /** @var \App\Models\OrganisationMember $member */?>
-        @foreach ($members as $member)
-            @if (empty($member->character))
-                @continue
-            @endif
-                <div class="font-extrabold">{{ $member->role }}</div>
-                <div>{!! $member->character->tooltipedLink() !!}</div>
-        @endforeach
-    </div>
+    <?php /** @var \App\Models\OrganisationMember $member */?>
+    @foreach ($members as $member)
+        @if (empty($member->character))
+            @continue
+        @endif
+        <div class="grid grid-cols-2 gap-2 members" data-role="{{ Illuminate\Support\Str::slug($member->role) }} data-status="  data-status="{{ $member->status_id }}">
```

Comment:
This will break people's custom css targetting the `members` class that I recently added. Please do
```
<div class="members flex flex-row gap-2">
@foreach()
   <div class="grid grid-cols-2 gap-2" data-role="..." data-status="">
     <div class="font-extrabold"></div>
     <div></div>
   </div>
@endforeach
</div>
```